manifests/server.pp

```diff
@@ -13,11 +13,15 @@
 # @param service_enable
 #   If `$service_name` should be enabled, see `enable` for `service` resource.
 # @param conf_dir
+#   Deprecated, use config_dir
+# @param config_dir
 #   Directory with clickhouse-server configuration.
 # @param conf_d_dir
-#   Directory with clickhouse-server included configuration.
+#   Deprecated, use config_d_dir
+# @param config_d_dir
+#   Directory with clickhouse-server included configuration. Unmanaged configs will be removed from the dirrectory during puppet running.
 # @param users_d_dir
-#   Directory with clickhouse-server users configuration.
+#   Directory with clickhouse-server users configuration. Unmanaged configs will be removed from the dirrectory during puppet running.
```

Comment:
Typo in `dirrectory` one `r` to much and I think it should be `... during puppet run` not `running`.